packages/pipeline/src/scripts/pull_slippage.ts

```diff
@@ -0,0 +1,49 @@
+import { Connection, ConnectionOptions, createConnection } from 'typeorm';
+
+import { logUtils } from '@0x/utils';
+
+import { EdpsSource} from '../data_sources/slippage';
+import { Slippage } from '../entities';
+import * as ormConfig from '../ormconfig';
+import { calculateSlippage } from '../parsers/slippage';
+import { handleError } from '../utils';
+
+// Number of orders to save at once.
+const BATCH_SAVE_SIZE = 1000;
+
+// USD amounts for slippage depths
+// tslint:disable-next-line:custom-no-magic-numbers
+const USD_AMOUNTS = [10, 100, 1000, 10000];
+
+// TODO: fetch from database
```

Comment:
We can do this as a followup if you like. If we keep it as a manual list, I think it should contain all the MM program tokens as well. So it's missing LOOM, DNT, CVC

Reply:
Yeah, if you don't mind, it would be great to ship a first version so that we start capturing data. Will add those 3 tokens.